demand-capacity-mgmt-backend/src/main/resources/application.yml

```diff
@@ -18,15 +18,26 @@
 #   SPDX-License-Identifier: Apache-2.0
 #   ********************************************************************************
 
+app:
+  openapi:
+    name: Demand and Capacity Management
+    description: Demand and Capacity Management project
+    version: 1
+    security:
+      enabled: ${SECURITY_ENABLE:true}
+      auth-url: ${keycloak.baseUrl}/realms/${keycloak.realm}/protocol/openid-connect/auth
```

Comment:
Same as below. Please add default values or instruct how to set these values, e.g. `.env` file

Reply:
We already have default values in the keycloak section at line no: 56

Reply:
Thanks for pointing that. As it did not work for me on windows, I thought this is the problem.

I consider this resolved.